README.md

```diff
@@ -1 +1,46 @@
-# BLS TypeScript Key Managment
+# BLS TypeScript Key Management
+
+![npm (tag)](https://img.shields.io/npm/v/bls-keygen/latest)
+[![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)](https://discord.gg/aMxzVcr)
+![GitHub](https://img.shields.io/github/license/chainsafe/bls-keygen)
+
+Utility methods for generating valid BLS keys from random bytes or mnemonic for NodeJs and Browser.
+
+Implementation is following EIPS: [EIP-2334](https://github.com/ethereum/EIPs/pull/2334), [EIP-2333](https://github.com/ethereum/EIPs/pull/2333)
+
```

Comment:
Could you link to the lower-level interface?
something like "For a low level hd wallet interface, see [@chainsafe/bls-hd-key](https://github.com/chainsafe/bls-hd-key)"